curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md

```diff
@@ -7,15 +7,15 @@ dashedName: task-25
 
 # --description--
 
-Before you learned to ask questions using `do,` you can also use the auxiliary verb to in negative sentences. In negative sentences you combine the auxiliary verb `do` (I, you, we, they) or `does` (he, she it) with the word `not` and place it right before the main verb:
+Before you learned to ask questions using `do,` you can also use it to create negative sentences by combining `do` (for I, you, we, they) or `does` (for he, she, it) with the word `not` and placing it right before the main verb:
```

Comment:
I'd suggest that the pronouns be surrounded by backslashes (e.g: `I`, `you`, `we`, `they`) in the parentheses.